Source/Core/Core/Core.cpp

```diff
@@ -913,7 +913,12 @@ void UpdateTitle(Core::System& system)
       g_video_backend->GetDisplayName(), Config::Get(Config::MAIN_DSP_HLE) ? "HLE" : "LLE");
 
   std::string message = fmt::format("{} | {}", Common::GetScmRevStr(), SSettings);
-  if (Config::Get(Config::MAIN_SHOW_ACTIVE_TITLE))
+
+  if (Config::Get(Config::MAIN_STATIC_TITLE))
+  {
+    message = "Dolphin Render Window";
```

Comment:
This should be translated. (Unless that would be a problem for people who swap between different interface languages, if they need the window title to be static across languages...? But then I would recommend "Dolphin" instead of "Dolphin Render Window".)

```suggestion
    message = GetStringT("Dolphin Render Window");
```

Reply:
Thanks for the feedback.
Yes I agree, it should be translated, I overlooked this.